### ISSUE TITLE
torch.nn 之 Normalization Layers - I Just Wanna Run

### ISSUE THREAD

**Issue by mmmwhy** [mmmwhy/gitalk-comment#4]
https://iii.run/archives/fae41911210f.html 

 torch.nn 是 torch 的神经网络计算部分，其中有许多基础的功能。本文主要记录一下 torch.nn 的 Normalization Layers。
